src/pybind/mgr/dashboard/frontend/src/app/ceph/cluster/create-cluster/create-cluster.component.html

```diff
@@ -44,6 +44,13 @@
         </div>
         <div *ngSwitchCase="'2'"
              class="ml-5">
+          <h4 class="title"
+              i18n>Create OSDs</h4>
+          <br>
+          <cd-osd-list [clusterCreation]="true"></cd-osd-list>
```

Comment:
Maybe add an attribute to hide the title, because showing it twice doesn't look nice. I guess the user knows that they are on the `Create OSD` page because the navigation on the left shows it. The same applies to the `Add Hosts` wizard page.

![Bildschirmfoto vom 2021-08-09 15-46-50](https://user-images.githubusercontent.com/1897962/128717273-dbc0711e-108d-4b78-a414-46344bb35a11.png)